classes/citation/CitationDAO.inc.php

```diff
@@ -83,49 +82,78 @@ function &getObjectById($citationId) {
 	}
 
 	/**
+	 * Import citations from a raw citation list to the object
+	 * described by the given association type and id.
+	 * @param $assocType int
+	 * @param $assocId int
+	 * @param $rawCitationList string
+	 */
+	function importCitations($assocType, $assocId, $rawCitationList) {
```

Comment:
What `assocType` is supported other than submissions?

Reply:
At the moment none. Shall I then maybe remove those columns for now and then later, if they are needed, we can add them?

Reply:
If you can think of other `assocType`s we might want then you can leave it in. Otherwise I'd be tempted to remove them. Your call.

Reply:
Thanks! I'll remove them for now... :-)